src/overlays/actors/ovl_Obj_Spinyroll/z_obj_spinyroll.c

```diff
@@ -28,121 +40,669 @@ const ActorInit Obj_Spinyroll_InitVars = {
     (ActorFunc)ObjSpinyroll_Draw,
 };
 
-// static ColliderTrisElementInit sTrisElementsInit[6] = {
-static ColliderTrisElementInit D_80A1F040[6] = {
+static ColliderTrisElementInit sTrisElementsInit[6] = {
```

Comment:
```suggestion
static ColliderTrisElementInit sTrisElementsInit[] = {
```
We avoid hard-coded numbers unless necessary